include/z64.h

```diff
@@ -1561,7 +1561,7 @@ struct GlobalContext {
     /* 0x18784 */ s32 (*damagePlayer)(struct GlobalContext* globalCtx, s32 damage);
     /* 0x18788 */ void (*talkWithPlayer)(struct GlobalContext* globalCtx, Actor* actor);
     /* 0x1878C */ void* unk_1878C; //! @TODO: Determine function prototype
-    /* 0x18790 */ void* unk_18790; //! @TODO: Determine function prototype
+    /* 0x18790 */ void (*unk_18790)(struct GlobalContext* globalCtx, s16 arg1, Actor* actor); //! @TODO: Determine function prototype
```

Comment:
```suggestion
    /* 0x18790 */ void (*unk_18790)(struct GlobalContext* globalCtx, s16 arg1, Actor* actor);
```
Shouldn't need the todo anymore